functionsLab.js

```diff
@@ -0,0 +1,296 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+let var1Num1;
+let var1Num2;
+const function1 = (num1, num2) => {
+  if(num1 === num2) {
+    return num2;
+
+  } else if(num1 > num2) {
+    return num1;
+
+  } else if(num1 < num2) {
+    return num2;
+
+  }
+}
+function1(5, 7);
+console.log("----------------------------------------------------")
+// ## Question 2
+//
+// Write a function named getDogAge that takes one argument: a dog's age in human years. The function returns the dog's age in dog years. The conversion rate is 1 human year to 7 dog years.
+//
+// Example:
+//
+// Input: `4`
+//
+// Output: `28`
+const getDogAge = (num) => {
+  return console.log("The age of your dog is: " + (num * 7));
+}
+getDogAge(7);
+console.log("----------------------------------------------------")
+// ## Question 3
+//
+// Write a function `arraySum` that takes in an array and returns the sum of all the elements.
+//
+// Example:
+//
+// Input: `[13,2,4,-2]`
+//
+// Output: `17`
+const arraySum = (inputArray) => {
+  let temp = 0;
+  for(let i in inputArray) {
+    temp += inputArray[i];
+  }
+  return console.log("The sum of the array is: " + temp);
+}
+let array3 = [1, 3, 5, 7, 9];
+arraySum(array3);
+console.log("----------------------------------------------------")
+// ## Question 4
+//
+// Write a function that takes in any two numbers and return their sum.
+//
+// Example:
+//
+// Input: `14, 5`
+//
+// Output: `19`
+const function4 = (num1, num2) => {
+  return console.log(num1 + num2);
```

Comment:
You can choose to either return the value or console.log the value but never both. This would console log the answer but you won't be able to assign value returned from the function into a variable since this function will return undefined.